### DISCUSSION TITLE
The deformation process cannot be displayed directly using paraview post-processing

### DISCUSSION THREAD

**Discussion by zengyuxin0807** [idaholab/moose#27906] [Q&A Modules: Solid mechanics]
Dear Moose Experts,

I am studying the example about the plasticity part of the crystal and now have some questions. When I run the exception.i file ( moose/modules/solid_mechanics/test/tests/crystal_plasticity/stress_update_material_based/exception.i) , I can't see the deformation over time like in the other examples when I put the results into the paraview software for visualisation.

![image](https://github.com/idaholab/moose/assets/169163866/4e810f99-a5c2-4b5a-b5a0-164cbcdcd1ba)

I want to get a deformation display like the other examples, how do I go about doing that?

I would greatly appreciate any insights or suggestions from the community to help resolve this issue.

Thank you very much for your time and consideration.

**Comment by GiudGiud** [comment]
Hello

I m not sure what you mean. Do you mean the displacements should be called disp instead of u ?

Guillaume

**Reply by zengyuxin0807** [reply]
Thank you very much for being able to reply to me, I'm sorry it took so long to get back to you, it's not the name of the displacement but how it shows up in the visualisation. As in the original question, you can see the shape change over time in the other examples, but not in the result of running the exception.i file.

**Reply by GiudGiud** [reply]
I think that's tied to the name.
You need paraview to recognize that the vector is a vector of the displacements of nodes
https://discourse.paraview.org/t/displacement-of-mesh-operator-in-paraview/11590/6

Either that or you can output the displaced mesh by specifying use_displaced_mesh in the Output

**Reply by zengyuxin0807** [reply]
Thank you very much for your answer, I will try this!
